gcp-bundle/build.gradle

```diff
@@ -26,6 +26,8 @@ project(":iceberg-gcp-bundle") {
   dependencies {
     implementation platform(libs.google.libraries.bom)
     implementation "com.google.cloud:google-cloud-storage"
+    implementation "com.google.cloud:google-cloud-bigquery"
```

Comment:
btw this requires updates to `LICENSE` and `NOTICE`

Reply:
I updated LICENCE and NOTIVE by using @bryanck 's gradle plugin